src/EPPlus/Table/PivotTable/Filter/ExcelPivotTableFilter.cs

```diff
@@ -378,6 +448,203 @@ internal void CreateValueFilter()
+                    case ePivotTableFilterType.NextWeek:
+                        DateTimeUtil.GetWeekDates(DateTime.Today.AddDays(7), out startDate, out endDate);
+                        return date.Value >= startDate && date.Value <= endDate;
+                    case ePivotTableFilterType.M1:
+                        return date.Value.Month == 1;
+                    case ePivotTableFilterType.M2:
+                        return date.Value.Month == 2;
+                    case ePivotTableFilterType.M3:
+                        return date.Value.Month == 3;
+                    case ePivotTableFilterType.M4:
+                        return date.Value.Month == 4;
+                    case ePivotTableFilterType.M5:
+                        return date.Value.Month == 5;
+                    case ePivotTableFilterType.M6:
+                        return date.Value.Month == 6;
+                    case ePivotTableFilterType.M7:
+                        return date.Value.Month == 7;
+                    case ePivotTableFilterType.M8:
+                        return date.Value.Month == 8;
+                    case ePivotTableFilterType.M9:
+                        return date.Value.Month == 9;
+                    case ePivotTableFilterType.M10:
+                        return date.Value.Month == 10;
+                    case ePivotTableFilterType.M11:
+                        return date.Value.Month == 11;
+                    case ePivotTableFilterType.M12:
+                        return date.Value.Month == 12;
+                    case ePivotTableFilterType.Q1:
+                        return date.Value.Month >= 1 && date.Value.Month <= 3;
+                    case ePivotTableFilterType.Q2:
+                        return date.Value.Month >= 4 && date.Value.Month <= 6;
+                    case ePivotTableFilterType.Q3:
+                        return date.Value.Month >= 7 && date.Value.Month <= 9;
+                    case ePivotTableFilterType.Q4:
+                        return date.Value.Month >= 10 && date.Value.Month <= 12;
+                    case ePivotTableFilterType.Yesterday:
+                        return date.Value.Date == DateTime.Today.AddDays(-1);
+                    case ePivotTableFilterType.Today:
+                        return date.Value.Date == DateTime.Today;
+                    case ePivotTableFilterType.Tomorrow:
+                        return date.Value.Date == DateTime.Today.AddDays(1);
+                }
+            }
+            return false;
+        }
+        internal bool MatchNumeric(object value)
+        {
+            var num = RoundingHelper.RoundToSignificantFig(ConvertUtil.GetValueDouble(value), 15); 
+            var value1 = RoundingHelper.RoundToSignificantFig(ConvertUtil.GetValueDouble(Value1, false, true), 15);
+            if (double.IsNaN(num) == false && double.IsNaN(value1) == false)
+            {
+                switch (Type)
```

Comment:
This should also have a default

Reply:
Unhandled enum values do not have a value (for example `LastMonth`), so default does nothing in this case.

Reply:
Above comment refer to `switch` starting on row 60 (not commented)
Added exception on `switch` starting on row 624.